setup.py

```diff
@@ -548,9 +549,38 @@ def hardcoded_custom_type_dep_constants(cls) -> str:
         return {**super().hardcoded_custom_type_dep_constants(), **constants}
 
 
+#
+# CapellaSpecBuilder
+#
+class CapellaSpecBuilder(MergeSpecBuilder):
+    fork: str = CAPELLA
+
+    @classmethod
+    def imports(cls, preset_name: str):
+        return super().imports(preset_name) + f'''
+from eth2spec.merge import {preset_name} as merge
+'''
+
+    @classmethod
+    def preparations(cls):
+        return super().preparations()
+
+    @classmethod
+    def sundry_functions(cls) -> str:
+        return super().sundry_functions()
+
+    @classmethod
+    def hardcoded_ssz_dep_constants(cls) -> Dict[str, str]:
+        constants = {
+            'FINALIZED_ROOT_INDEX': 'GeneralizedIndex(105)',
+            'NEXT_SYNC_COMMITTEE_INDEX': 'GeneralizedIndex(55)',
+        }
+        return {**super().hardcoded_ssz_dep_constants(), **constants}
```

Comment:
No needs to implement these methods if there are no new settings. It will just use `MergeSpecBuilder` methods. 


```suggestion
```

Reply:
I was getting some build errors without it so added them. But will try again

Reply:
annoying. I tried again and it worked!
thanks!